client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-pattern-list.tsx

```diff
@@ -13,27 +14,30 @@ interface Props {
 	onDoneClick: () => void;
 }
 
-const ScreenPatternList = ( { selectedPattern, onSelect, onBack, onDoneClick }: Props ) => {
+const ScreenPatternList = ( { selectedPattern, onSelect, onDoneClick }: Props ) => {
 	const translate = useTranslate();
 	const patterns = useSectionPatterns();
 
 	return (
 		<>
-			<NavigatorHeader title={ translate( 'Add patterns' ) } />
+			{ ! isEnabled( 'pattern-assembler/categories' ) && (
```

Comment:
I guess we don't need to check this as we won't use the `ScreenPatternList` when the flag is enabled.